concent_api/core/tests/test_core_views.py

```diff
@@ -8,11 +8,12 @@
 from django.http                    import JsonResponse
 from django.utils                   import timezone
 
+from golem_messages                 import message
 from golem_messages                 import settings
+from golem_messages.factories       import tasks
+from golem_messages.message         import Message as GolemMessage
```

Comment:
This is not needed, `message` is already imported so you can use it as `message.Message`